recipe/meta.yaml

```diff
@@ -62,10 +62,15 @@ outputs:
         - {{ compiler("cxx") }}
         - {{ stdlib("c") }}
         - cmake
+        - numpy                               # [build_platform != target_platform]
+        - python                              # [build_platform != target_platform]
```

Comment:
I think you might need to add the cross_python dep as well if you're going to use python? see the rdkit package and https://conda-forge.org/docs/maintainer/knowledge_base/#details-about-cross-compiled-python-packages